base/applications/mmc/console.c

```diff
@@ -334,6 +337,16 @@ FrameOnCommand(HWND hwnd,
             PostMessage(hwnd, WM_CLOSE, 0, 0);
             break;
 
+        case IDM_HELP_ABOUT:
+            if (AllocAndLoadString(&lpTitle, hAppInstance, IDS_APPTITLE))
+            {
+                hIcon = LoadIcon(hAppInstance, MAKEINTRESOURCEW(IDI_MAINAPP));
+                ShellAbout(NULL, lpTitle, NULL, hIcon);
```

Comment:
```suggestion
                ShellAbout(hwnd, lpTitle, NULL, hIcon);
```

Reply:
This does indeed seem like the mmc.exe behavior in windows.